src/main/encryptors/serialization.ts

```diff
@@ -20,7 +20,7 @@ export function deserialize(bytes: Uint8Array): any {
 function convertJsonBuffers(obj: any): any {
   if (isJsonBuffer(obj)) {
     return Buffer.from(obj)
-  } else if (typeof obj === "object") {
+  } else if (obj !== null && typeof obj === "object") {
```

Comment:
Since our new behavior is to collect values to be encrypted into a JS object, we have to handle the potential presence of `null` in that object now.